src/tag.rs

```diff
@@ -1425,4 +1448,38 @@ mod tests {
         tag_file.seek(io::SeekFrom::Start(0)).unwrap();
         assert!(!Tag::remove_from(&mut tag_file).unwrap());
     }
+
+    #[test]
+    fn aiff() {
```

Comment:
Thank you for including a test! :D

Could you rename this test to something like `aiff_read_and_write`? That makes it more descriptive in the test output if it ever fails